index.js

```diff
@@ -228,9 +228,11 @@ module.exports = {
     "prefer-const": 2,
     "prefer-reflect": 0,
     "prefer-spread": 0,
-    "prefer-template": 0,
+    "prefer-template": 2,
     "require-yield": 0,
+    "keyword-spacing": 2,
```

Comment:
Requires spacing around keywords, makes the following an error:

```javascript
if (foo) {
    //...
}else if (bar) {
    //...
}else {
    //...
}
```

see https://eslint.org/docs/rules/keyword-spacing for more info